bagit.xml

```diff
@@ -477,10 +477,12 @@ placeholder file with a name such as ".keep".
             checksum algorithm.
           </t>
           <t>
-            A bag &may; contain one or more tag manifests.
+            A bag &may; contain one or more tag manifests, in which case each tag manifest &must; list the same set of tag files.
```

Comment:
Looking at this again, do we want to specify `&must;` here? Seems like a burden to an implementor without much benefit. Instead should we switch it to `&should;`?